doc-examples/azure-cosmos-example-kotlin/src/main/kotlin/example/PersonRepository.kt

```diff
@@ -96,10 +97,12 @@ interface PersonRepository : CrudRepository<Person, String>, JpaSpecificationExe
         // end::where[]
 
         // tag::or[]
-        fun nameOrAgeMatches(age: Int, name: String?) = where<Person> {
-            or {
-                root[Person::name] eq name
-                root[Person::age] lt age
+        fun nameOrAgeMatches(age: Int, name: String?) = query<Person> {
```

Comment:
Please don't change existing tests

Reply:
sure!

Reply:
I still see that the previous tests are modified

Reply:
Oh I missed that one, will fix

Reply:
done